### PR TITLE
Create fibonacci-generator.sublime-snippet

### PR DESCRIPTION
**Pull request by KleinerNull**
Here another fibonacci snippet, because we need more fibs, the most useful number series!

### CODE REVIEW

File: Python/fibonacci-generator.sublime-snippet [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<snippet>
+    <content><![CDATA[
+def fibonacci_generator(a=0, b=1):
+    while True:
+        yield a
+        a, b = b, a + b
+]]></content>
+    <tabTrigger>fibonaccinumbers</tabTrigger>
```
Comment (yasintoy) [inline Python/fibonacci-generator.sublime-snippet]: can you change this trigger as fibonaccigenerator

**Comment by yasintoy** [comment]
thanks, good job.

**Comment by yasintoy** [comment]
by the way, we must add SnipLime to Vim and Pycharm.

**Comment by KleinerNull** [comment]
Personally I don't use Sublime, I like Atom and jupyter more ;)

**Pull request merged**